docs/BEMAN_STANDARD.md

```diff
@@ -112,27 +111,35 @@ to build it, and links to further documentation.
 
 ## `README.md`
 
-**[README.TITLE]** RECOMMENDATION: The `README.md` should begin with a level 1
-header with the name of the library optionally followed with a ":" and short
-description.
+**[README.TITLE]** RECOMMENDATION: The `README.md` must begin with a level 1 header with the name of the library optionally followed with a ":" and short description.
 
 Examples:
 
 ```markdown
 # beman.sender_receiver: Scalable Asychronous Program Building Blocks
 ```
 
-**[README.PURPOSE]** RECOMMENDATION: Following the title, the `README.md` should
-contain a one- or two-paragraph summary describing the library's purpose.
+**[README.PURPOSE]** REQUIREMENT: Following the title, the `README.md` must contain a one or two-paragraph summary describing the library's purpose.
 
-**[README.IMPLEMENTS]** RECOMMENDATION: Following the purpose and a newline, the
-`README.md` should indicate which papers the repository implements. Use the following style:
+**[README.IMPLEMENTS]** REQUIREMENT: Following the purpose and a newline, the `README.md` must indicate which papers the repository implements. Use the following style:
```

Comment:
```suggestion
**[README.IMPLEMENTS]** REQUIREMENT: Following the purpose and a newline, the `README.md` must indicate which papers the repository implements. Use the following style:
```

I think beginning the sentence with `Following the X, Y, Z` servers no purpose. I would mention per each section what it should contain

Reply:
Again, this is an existing rules, I just change whitespace. If you would like to rewrite all senteces from BEMAN_STANDARD, it should be a new PR.

And I disagree, we want to enforce the exact formating. Each section already has these mentions, this current enumaration shows how to position this section after others.